src/main/webapp/js/store.js

```diff
@@ -1,16 +1,45 @@
 import ReduxPromise from 'redux-promise';
 import reduxThunk from 'redux-thunk';
 import { createStore, applyMiddleware, compose } from 'redux';
-import { offline } from '@redux-offline/redux-offline';
+import { createOffline } from '@redux-offline/redux-offline';
 import offlineConfig from '@redux-offline/redux-offline/lib/defaults';
 
 import rootReducer from './reducers/index';
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
I think you can remove it for now, push to a separate branch.
I created a separate ticket for this: https://applab.atlassian.net/browse/NEWEBODAC-77